src/login/login.php

```diff
@@ -1,18 +1,26 @@
 <?php
     session_start();
 
+    require_once( realpath( dirname( __FILE__ ) ) . "/../php/DAL.php" );
+
+    $dal = new DAL();
+    $user_id = $dal->query("SELECT user_id FROM User WHERE username='$username', password='$password'");
     $username = $_POST['user1'];
     $password = $_POST['password1'];
 
     $_SESSION[ "user_id" ] = "3";
```

Comment:
not sure if this session variable should stay equal to 3 or changed to something else